src/js/main.js

```diff
@@ -0,0 +1,225 @@
+let todoList= [];
+let storage = localStorage;
+let count=0;
+
+class Todo{
+  constructor(id, name, completed)
+  {
+    this.id = id+"";
+    this.name = name;
+    this.completed = completed;
+  }
+}
+
+window.onload= function(){
+    if(storage.getItem("list")===null) return;
+
+    const storageTodo = storage.getItem("list");
+    const list = JSON.parse(storageTodo);
+    count = Number.parseInt(storage.getItem("size"));
+    
+    list.forEach(todo =>
+        {
+        todoList.push(new Todo(todo.id, todo.name, todo.completed));
+    });
+    showItem();
+}
+
+let inputElement = document.getElementById("new-todo-title");
+
+function enterkey(){
+  if(window.event.key == 'Enter'){
+    if(inputElement.value.trim()==""||null||undefined||NaN)
+    {
+      alert("빈 값입니다."+inputElement.value.trim());
+      return ; 
+    }
+    addItem(inputElement.value);
+  }
+  inputElement.innerHTML.value ="";
+}
+
+function addItem(value)
+{
+  count++;
+  const todo =  new Todo(count, value, false);
+  todoList.push(todo);
+  storage.setItem("list", JSON.stringify(todoList));
+  storage.setItem("size", count+"");
+  showItem();
+  inputElement.value="";
+}
+
+function showItem(){
+    let list ="";
+    for(let i of todoList){
+        let com = i.completed == true ? "completed" :"";
```

Comment:
i.complete가 이미 boolean이기 때문에 `== true`로 값 비교를 하기 보단 그 자체를 조건으로 쓰면 더 좋을듯 합니다!

```suggestion
        let com = i.completed ? "completed" : "";
```

Reply:
@baekdev 감사합니다 수정했습니다 👍 👍 
fixed in 940faaf.